recipes/implot/all/test_package/conanfile.py

```diff
@@ -1,15 +1,31 @@
-from conans import ConanFile, CMake, tools
 import os
 
-class TestPackageConan(ConanFile):
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.build import can_run
+
+
+class ImplotTestConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake", "cmake_find_package_multi"
+
```

Comment:
```suggestion
    generatos = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
    test_type = "explicit"
```

Reply:
```suggestion
    test_type = "explicit"
```

Reply:
test type would be nice to add

Reply:
Done in [23bca16](https://github.com/conan-io/conan-center-index/pull/16847/commits/23bca1645782c7804ec2ff131f8a0f459de36879)